src/Promitor.Integrations.Sinks.Statsd/Configuration/StatsdFormatterTypes.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Promitor.Integrations.Sinks.Statsd.Configuration
+{
+    public class StatsdFormatterTypes
+    {
+        public const string DEFAULT = "default";
+        public const string CUSTOM = "custom";
```

Comment:
Since it's backed in and not configurable, we might as well just call this Geneva